src/styles/app.styl

```diff
@@ -470,6 +470,7 @@ footer
     display flex
     justify-content center
     align-items stretch
+    margin-top 27px
```

Comment:
@GoOz : is it ok to have a dimension in pixels here ? Where does `27` comes from ?